_posts/2019-09-02-shorter-isn't-better.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: post
+title: "Shorter functions are't always better"
+date: 2019-09-05 10:00:00 +0800
+categories: blog
+author: Lex Huang
+---
+
+# Shorter functions aren't always better
+
+This is just a thought that came to me during day to day code reviews and I want to share the idea with you. My point is that for functions, readability, understandability and maintainability is much more important the length and the abstraction level.
```

Comment:
Let's rephrase it a bit:
This is an idea that came to me during day to day code reviews that I want to share with you. A function's readability, understandability, and maintainability are more important than the length and abstraction level of the function.